webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -658,6 +659,17 @@ var userOverride = {
             document.addEventListener('mousedown', function(e) {
                 document.body.classList.remove("accesibility");
             });
+
+            // the browser call getVoices is async
+            // so we are updating the array whenever they're available
+            if (VOICE_OUTPUT_ENABLED && "speechSynthesis" in window) {
+                AVAILABLE_VOICES_FOR_TTS = speechSynthesis.getVoices();
+                if (speechSynthesis.onvoiceschanged !== undefined) {
+                    speechSynthesis.onvoiceschanged = function () {
+                        AVAILABLE_VOICES_FOR_TTS = speechSynthesis.getVoices();
```

Comment:
I am just concerned for following:
1. browser compatibility is limited
![Screenshot 2021-02-13 at 10 19 58 AM](https://user-images.githubusercontent.com/19753380/107841867-08f2cb80-6de5-11eb-9576-a1bd6591017e.png)
2. It is an experimental function.

Is it safe to use ?

Reply:
Yeah, this will cover the cases when `onvoiceschanged` isn't available.